Resources/Prototypes/Recipes/Construction/Graphs/furniture/fires.yml

```
@@ -0,0 +1,49 @@
+- type: constructionGraph
+  id: FiresGraph
+  start: start
+  graph:
+    - node: start
+      actions:
+        - !type:DestroyEntity {}
+      edges:
+        - to: BonfireNode
+          completed:
+            - !type:SnapToGrid { }
```

YAMLlint (1.35.1)

[error] too many spaces inside empty braces

(braces)